chaosaws/asg/probes.py

```diff
@@ -271,7 +271,7 @@ def get_asg_by_name(asg_names: List[str],
     invalid_asgs = [a for a in asg_names if a not in valid_asgs]
 
     if invalid_asgs:
-        raise FailedActivity(f'No ASG(s) found matching: {invalid_asgs}')
+        raise FailedActivity('No ASG(s) found matching: %s' % invalid_asgs)
```

Comment:
Any reason you changed this one?

I tend to use the third option:

```
raise FailedActivity('No ASG(s) found matching: {}'.format(invalid_asgs)
```

I had actually missed the use of the f'...' litteral here.

Reply:
Would you prefer I use the `.format()` option over C-style? 

I use that option as it's the one I've used as long as I can remember... old habits.

Reply:
I used to use the C-style as well (we must be around the same age ;)). But I moved to .format to avoid some (rare) pitfalls with them.

If it's not too much trouble to update, then that'd be welcome. And likely added to a contributor's guide on our doc (a task for me to carry).

Reply:
No trouble at all. I'll update the formatting shortly.